lib/Conversion/TorchToTcp/Utils.cpp

```diff
@@ -389,3 +382,101 @@ Value torch_to_tcp::scalarToTcpTensor(ConversionPatternRewriter &rewriter,
   return rewriter.create<tensor::FromElementsOp>(op->getLoc(), targetType,
                                                  ArrayRef<Value>{scalarValue});
 }
+
+void TorchToTcpCustomOpConversionHelper::addOperand(std::string opName,
+                                                    Value value) {
+  if (conversionResult.failed())
+    return;
+  operandNames.push_back(opName);
+  operands.push_back(value);
+}
+
+void TorchToTcpCustomOpConversionHelper::addAsMultipleTensorOperands(
+    std::string opNamePrefix, mlir::Value value) {
+  if (conversionResult.failed())
+    return;
+  mlir::SmallVector<Value> indicesTorchType;
+  if (!torch::Torch::getListConstructElements(value, indicesTorchType)) {
+    conversionResult = op->emitError(
+        "unimplemented: the tensor list is not from list construct");
+    return;
+  }
+
+  mlir::SmallVector<Value> indexTensors = torch::Torch::getTypeConvertedValues(
+      rewriter, op->getLoc(), typeConverter, indicesTorchType);
+
+  for (size_t i = 0; i < indexTensors.size(); ++i) {
+    addOperand(opNamePrefix + std::to_string(i), indexTensors[i]);
+  }
```

Comment:
Can we make this conditional on `indexTensors.size() > 1` (to avoid the single element list case having a redundant `_0` suffix)